daisy_workflows/image_build/sqlserver/sql_install.ps1

```diff
@@ -296,18 +296,53 @@ function Install-SSMS {
     Write-Host "Not installing SSMS for config ${sql_server_config}"
     return
   }
-  Write-Host 'Installing SSMS'
+  
+  Write-Host 'Installing SSMS from layout'
+   
+  $SSMS_SOURCE_ZIP = "C:\SSMS_Installer.zip"
```

Comment:
How is it obtaining this file? We were obtaining the exe from gcs, however the gsutil call is being removed.